processor/src/test/kotlin/com/tobrun/datacompat/DataCompatProcessorTest.kt

```diff
@@ -15,14 +15,16 @@ private val simpleTest = SourceFile.kotlin(
 """
 import com.tobrun.datacompat.annotation.DataCompat
 
+interface EmptyInterface
 /**
  * Represents a person.
  * @property name The full name.
  * @property nickname The nickname.
  * @property age The age.
  */
+@Deprecated
 @DataCompat
-private data class PersonData(val name: String, val nickname: String? = null, val age: Int)
+private data class PersonData(val name: String, val nickname: String? = null, val age: Int) : EmptyInterface
```

Comment:
it will be nice to add several interface

Reply:
done